docs/SOPs/update_ena_runs_SOP.md

```diff
@@ -0,0 +1,99 @@
+   ```
+2. Install `gsutil` in your environment in the EBI cluster and log in using your EMBL-EBI google account.
+   You could follow instructions from https://cloud.google.com/storage/docs/gsutil_install to install `gs_util`. 
+   See more details [here](https://ebi-ait.github.io/hca-ebi-dev-team/admin_setup/Setting-up-access-to-Terra-staging-area.html#using-your-google-account) about setting up your access to Terra staging area.
+3. Get DSP's Webin [credentials](https://console.aws.amazon.com/secretsmanager/home?region=us-east-1#!/secret?name=ingest%2Fwebin-creds). (Only an ingest developer has access to this atm)
+4. Clone the ingest-archiver repository. The scripts that will be used is in `ena` directory of that repo.
+   ```
+   git clone https://github.com/ebi-ait/ingest-archiver.git
+   pip install -r requirements.txt
+   ```
+5. Get your JWT Token from Ingest UI.
+   1. Log in in Ingest UI https://contribute.data.humancellatlas.org/login using the account which has the WRANGLER role.
+   1. In Chrome, right click and select _Inspect_ to open the developer console. Select the Network tab.
+   1. Refresh the page, `https://contribute.data.humancellatlas.org/home`
+   1. Check the _Authorization_ in headers of the request to `https://api.ingest.archive.data.humancellatlas.org/auth/account`
+   1. Copy the token after _Bearer_ prefix: 
+      ```
+      Authorization: Bearer <copy the very long string of random characters>
+      ```
+   1. That token has 1 hr validity. The token will be needed in the submitter script later.
+## Steps
+
+1. Get the list of sequencing runs to be suppressed. This can be downloaded as TSV/JSON from the ENA Browser. 
+   
+2. Make sure
+   that the metadata in Ingest contains sequencing experiment accessions. The submitter script will raise an error if
+   any of the assay processes has no accession. The assay processes in the submission should have the following
+   property:
+
+    ```
+    "insdc_experiment": {
+      "insdc_experiment_accession": "ERX4319109"
+    }
+    ```
+
+2. Clear the sequencing run accessions in file metadata. The following should not be in the file metadata json:
+    ```
+    "insdc_run_accessions": [
+      "ERR6449905"
+    ]
+    ```
+   
+   Update script to have a jwt token from the Ingest UI then run the following: 
+    ```
+    python clear_run_accession_from_files.py <submission-uuid>
+    ```
+
+4. Download all files from Ingest / Terra upload area to any directory inside `/nfs/production/hca/` in the EBI cluster.
+   The files may also be in the hca-util upload area but we should make sure they're valid. Using Ingest/Terra upload area means the files have already been validated before.
+   Please prefer downloading the Terra upload area as downloading from Ingest upload area will incur cost to our AWS account.
+
+5. Checksum all the files.
+    ``` 
+    $ cd to directory where you've downloaded the files 
+    $ md5sum * > <md5-filename>.txt
+    ```
```

Comment:
```suggestion
    ``` 
    gsutil hash -hm gs://broad-dsp-monster-hca-prod-ebi-storage/prod/<project_uuid>/data/* | grep -A1 "hex" | awk -F"/" '{printf $4 $1}' | awk -F"--" '{for (i=1;i<NF;i++)print $i}' | awk -F":" '{print $1 $3}' > <md-filename>.txt
    ```
```

it's a bit long but with this command you get the md5's instantly